CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -339,11 +339,15 @@ protected function setFieldMetadata(): void {
    */
   public function getImportEntities() : array {
     $softCreditTypes = ContributionSoft::getFields()
-      ->setLoadOptions(['id', 'name', 'label', 'description'])
+      ->setLoadOptions(['id', 'name', 'label', 'description', 'is_default'])
```

Comment:
is there a reason why this `getFields` call has permission checks enabled? Seems like it should be FALSE

Reply:
@colemanw I've changed it - presumably permissions on soft credits -=== permissions on contribution & hence anyone who can import contributions can get soft credit info - but OTOH permissions can pop up problems